torch/_inductor/ir.py

```diff
@@ -3848,7 +3835,13 @@ def codegen(self, wrapper):
             # in C++ wrapper, we don't pass constexpr args, as they don't
             # get added as parameters to the PTX code compiled from the
             # user-defined Triton kernel (only non-constexpr args do)
-            args = self._remove_constexpr_args(args)
+            from triton.runtime.jit import JITFunction
```

Comment:
You dont need any of this, it is already a JITFunction

kernel, configs = self.get_kernel_and_configs()

took care of it for you

Reply:
3839-3843 are not needed

Reply:
Nope: autotuning tests fail. There `kernel` is an `Autotuner` instance which doesn't have `constexpr`. Need to unwrap. That's why I updated the PR.